### ISSUE TITLE
Hard to read module names

### ISSUE THREAD

**Issue by sporto** [dillonkearns/graphqelm#21]
We have a mutation named `BillingSubscriptions_UpdateBillingEmails`
This is because our API is pretty big and we need to namespace things.

e.g. something like this in Ruby

```
module Mutation
   module BillingSubscriptions
      UpdateBillingEmails = Mutation....
```

We name "BillingSubscriptions_UpdateBillingEmails" for graphQL.

The payload in the introspection schema looks like:

```
           "type": {
              "kind": "OBJECT",
              "name": "BillingSubscriptions_UpdateBillingEmailsPayload",
              "ofType": null
            },
```

When graphqelm generates the Elm modules we end up with

```
Api.Object.BillingsubscriptionsUpdatebillingemailspayload
```

`BillingsubscriptionsUpdatebillingemailspayload` is a pretty hard name to parse mentally. It would be great if graphql would respect the camel case and do something else with the `_`.  It don't suppose underscore are valid in a module name. Just `BillingSubscriptionsUpdateBillingEmailsPayload` would be better.

**Comment by dillonkearns** [comment]
Thanks for the issue @sporto, it's really helpful to see this use case. It appears that non-leading underscores are valid in Elm module names. I'll go ahead and preserve them, and go with the minimal-change normalization strategy we discussed (just move leading underscores to the tail-end, and make sure the first letter's case is correct).

**Comment by dillonkearns** [comment]
Hey @sporto, I published a fix for this and #19.

To make the diff more manageable, I released npm version `2.0.3` with the changes to remove the comments with the version number data and instead use the metadata file, and a separate npm version, `2.1.0`, which changes the name normalization strategy. I would recommend doing the `2.0.3` update first to remove some noise from your pull request.

You can see [`NormalizeTests.elm`](https://github.com/dillonkearns/graphqelm/blob/master/tests/Generator/NormalizeTests.elm) for examples of how the normalization works.

Let me know how the update goes!

**Comment by sporto** [comment]
This is fixed now, thanks. Closing.
